pages/index.js

```diff
@@ -1,3 +1,5 @@
+import { useState, useEffect } from 'react'
```

Comment:
This file has changes on every line which makes it difficult to review and to merge (conflicts with other prs). Could you please move your newly added code to another file and import it here? You could use the `/sections/` or `/components/` folder, depending on what you're adding.

I think the reason everything has changed is that you are using `useIsScrolledFromTop`, but I believe it would work if you created a separate component and used the hook there.

Reply:
I've created a new section and pulled both the HTML and CSS into new files.